library/dokku_network_property.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# from ansible.module_utils.basic import *
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_network_property
+short_description: Set or clear a network property for a given dokku application
+options:
+  app:
+    description:
+      - The name of the app
+    required: True
+    default: null
+    aliases: []
+  network:
```

Comment:
Looking at [how `network:set` works](https://dokku.com/docs/networking/network/#attaching-an-app-to-a-network), I think it would be cleaner to call this variable `value`.

The network name just so happens to be the value for the three properties you have chosen to support here